src/logger/index.ts

```diff
@@ -0,0 +1,11 @@
+
+export const info = console.log
+export const warn = console.warn
+export const debug = console.log // TODO only when dev or debug flag on
+export const error = console.error
+
+export const makeLogger = (ns: string) =>
+  ({ info: (a: any) => info(`[${ns}]`, a),
+     warn: (a: any) => warn(`[${ns}]`, a),
+     debug: (a: any) => debug(`[${ns}]`, a),
+     error: (a: any) => error(`[${ns}]`, a), })
```

Comment:
add logger separately, just defers to console.log for now, but will let us change it later.  i know this should be in separate file